### ISSUE TITLE
Switching from group to single player via airplay

### ISSUE THREAD

**Issue by xcracing** [philippe44/LMS-Groups#20]
Love your software, I use it daily to do exactly what I want with my audio system in my home and home office.   You've really helped keep the LMS ecosystem up with the times!

I have several of your packages for LMS installed.   In this case, the issue involves the ShairTunes2 fork and the LMS-Groups plugins.

My scenario is this - I have LMS setup on a RPi.    I have some native Squeezebox players as well as a couple RPi's running PiCorePlayer.     I have used LMS-Groups to set up various groups withing the house (this plugin plugs a HUGE hole in the LMS sync system that annoyed me for years!!).   Some groups overlap (For instance, I have a "whole house" group - then I have a separate Bedroom and Bath group that has two of the same players).     

I generally use my iPhone & Airplay to select the groups or individual players and play music from various services.

Generally this works very well, with a couple small issues here and there.    The issue I'm logging here is that sometimes after playing to a group, then trying to switch to an individual player (contained in the group), the whole group plays instead of just the individual player - like the synchronization isn't being cancelled.

The only way I've found to resolve this is to go into LMS and individually click the power button on the players that should no longer be playing.     Then, I can can switch back to the group, the players will all work - then switch back to the individual player and it will work as expected.

Not sure how to duplicate but it happens fairly often.

**Comment by philippe44** [comment]
Thanks :-)
Do you pause the group before switching or do you simply select another "target" in your iDevice? I will take a better look but the issue might be that I need a "stop/pause" command on the group to disassemble it and then a "play" on the new player. I'm wondering if there might be some kind of race condition where LMS command are queued in a way that the group is not yet disassembled when the play command is received. Can you take some Shairtunes and Group Player logs? Put them both on "info"

**Comment by xcracing** [comment]
Many times I just switch groups but the most recent time I noticed, the players had been idle for a while.   I don't remember how their last play session was terminated - sometimes I just leave the house and the range of the wifi obviously makes the playback stop.   Sometimes I just swipe the app (i.e. pandora usually) away to stop the music.

I've got logging set to info on both, I'll find them and post next time the issue arises.  Also, next time I'll see if pausing, switching, then playing resolves the issue.
